src/runner.py

```diff
@@ -229,7 +230,12 @@ def remove(self, remove_token: str) -> None:
             self.status.runner_id,
             self.config.path.path(),
         )
-        self._clients.github.delete_runner(self.config.path, self.status.runner_id)
+        try:
+            self._clients.github.delete_runner(self.config.path, self.status.runner_id)
+        except HTTPError:
+            logger.exception("Unable the remove runner on GitHub: %s", self.config.name)
```

Comment:
Would it help to log the exception as well?
```suggestion
        except HTTPError as exc:
            logger.exception("Unable the remove runner on GitHub: %s, exception: %s", self.config.name, exc)
```

Reply:
The `logger.exception` would log the stack trace of the exception by default.